src/subquery.js

```diff
@@ -0,0 +1,43 @@
+import _ from 'lodash/fp'
+import { getTypePropOrError } from './types'
+
+// A subquery (in contexture-client) is about taking the output of one search and makng it the input for another search.
+// This is an in memory, cross-database, "select in" join on sources that don't need to be relational.
+export default _.curry((types, from, fromPath, to, toPath) => {
+  let toNode = to.getNode(toPath)
+  let fromNode = from.getNode(fromPath)
+
+  // Dispatch on toNode when fromNode is markedForUpdate to trigger loading indicator immediately on toNode when fromNode updates
```

Comment:
I like `fromNode` and `toNode`. Can we use something like that instead of `get` and `use` (I know I made a similar comment above, it's just for emphasis)?

Reply:
They're different concepts - get/use is about retrieving and use the join values. The name would be getSubqueryFromNodeValues and useSubqueryFromNodeValuesInToNode - from/to doesn't obviate the need to specify get/use. Alternatively we could have `getSubqueryValues` and `buildSubqueryMutation` but that seems more obtuse. Use also seemed clearer then set, espcially since it doesn't actually _perform_ the mutation.